runner/src/server/views/pay-error.html

```diff
@@ -6,20 +6,29 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     {% set tmpl = 'There was a problem with your payment' %}
-
     {{ govukErrorSummary({
       titleText: tmpl,
       errorList: [
-        { text: "The payment failed" }
+        { text: "The payment failed"  }
       ]
     }) }}
+
+    {% if customPayErrorMessage %}
+    <p class="govuk-body">
+      {{ customPayErrorMessage | safe }}
+    </p>
+    {% endif %}
+
```

Comment:
Alternatively we could make this an if...else condition, idk what's more preferable (my heart tells me default value)

Reply:
Neither, for now. We'll match existing feature set. (No additional message)